coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/basic/ProposalDevelopmentHomeController.java

```diff
@@ -293,7 +293,8 @@ public ModelAndView docHandler(@ModelAttribute("KualiForm") DocumentFormBase for
                propDevForm.setDefaultOpenTab(defaultOpenTab);
            }
 
-            if (StringUtils.equals(form.getRequest().getParameter("viewDocument"),"true") || StringUtils.equals(viewOnly, "true")) {
+            if (!propDevForm.getDocument().getDocumentHeader().getWorkflowDocument().isEnroute()
+                    && (StringUtils.equals(form.getRequest().getParameter("viewDocument"),"true") || StringUtils.equals(viewOnly, "true")) ) {
```

Comment:
This is weird. I think I can see why we need to do this, but I'd love it if we can not do this in the future. :)